pages/header.py

```diff
@@ -0,0 +1,53 @@
+from enum import verify
+
+from constants.header import HeaderConsts
+from pages.base_page import BasePage
+from pages.utils import wait_until_ok
+
+
+# from pages.utils import log_wrapper
+
+
+class Header(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.const = HeaderConsts
+
+    @wait_until_ok(timeout=3, period=0.5)
+    def click_and_validate_create_post_button(self):
```

Comment:
Why do you need this verification ?
Anyway this button won't disappear as header is always exists.